grafana-plugin/src/containers/Labels/Labels.tsx

```diff
@@ -30,6 +31,13 @@ const Labels = observer(
 
     const { labelsStore } = useStore();
 
+    const onChange = (value: LabelKeyValue[]) => {
+      if (onDataUpdate) {
+        onDataUpdate(value);
+      }
+      setValue(value);
+    };
+
     useImperativeHandle(
       ref,
```

Comment:
do we still the need ref if we pass data changes outside through `onDataUpdate`?

Reply:
`IntegrationForm` relies on it still. I didn't want to refactor too much outside webhook to not create conflicts. I suggest we look into it with @maskin25 , I can explain the approach I used here instead of using ref and either me or Maxim will introduce it in IntegrationForm in separate PR